### DISCUSSION TITLE
How to change text size while viewing epubs in Sumatra PDF 3.4.6

### DISCUSSION THREAD

**Discussion by ckush** [sumatrapdfreader/sumatrapdf#3617] [Q&A] [answered]
I am trying to read an epub and I remember earlier in Sumatra you could change the text size for epubs, now all I can do is zoom in like a pdf and the default size is too small (see screenshot). On my laptop I can barely read anything even in full screen mode.
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/25913086/a76c23ee-27c0-415c-8ec2-2de9e47a396c)
My question is how can I increase the text size without zooming in. I don't want to zoom in because then I can't move between pages through the arrow keys and I also have to move around on each page.

**Comment by GitHubRulesOK** [comment, marked as answer]
The TextSize facility was part of SumatraPDF "Legacy" viewer which used "reflowable" text feature of earlier eBook formats , 3.4.6 was "updated" to the "fixed" layout and relies more on the font size set by the Author. There are open requests for that to be changed https://github.com/sumatrapdfreader/sumatrapdf/issues/3382 but it is not a small task
